examples/react-datastore/.graphqlrc.yml

```diff
@@ -0,0 +1,8 @@
+schema: './src/schema.graphql'
+extensions:
+  graphback:
+    # path to data mode file(s)
+    model: './src/model/runtime.graphql'
+    plugins:
+      offix-datasync-client-plugin:
```

Comment:
I think it is best for us to also build schema for developers to see/share etc.

Reply:
Can we take out the schema reference? I tried removing it and got some errors

Reply:
This goes against this file
This file is build to describe schema. 
You can get schema from link if you wish.... 

But generally we should generate visible schema for users - drop schema plugin explicitly and dump schema somewhere.